src/realtimecontroller/RealtimeState.ts

```diff
@@ -102,3 +96,11 @@ export default class RealtimeState {
    */
   receiveDataMessageCallbacks: Map<string, ((dataMessage: DataMessage) => void)[]> = new Map();
 }
+
+export type VolumeIndicatorCallback = (
```

Comment:
As a pattern we have not exported types from the same file in the SDK. You can check [RealtimeVolumeIndicator](https://github.com/aws/amazon-chime-sdk-js/blob/master/src/realtimecontroller/RealtimeVolumeIndicator.ts) or [RealtimeAttendeePositionInFrame](https://github.com/aws/amazon-chime-sdk-js/blob/master/src/realtimecontroller/RealtimeAttendeePositionInFrame.ts). You can export a separate class (which is the pattern followed elsewhere) in the [realtimecontroller](https://github.com/aws/amazon-chime-sdk-js/tree/master/src/realtimecontroller) folder and as a convention follow the same naming pattern: `RealtimeVolumeIndicatorCallback`. @simmkyu @ltrung any suggestions?

Reply:
That seems like a reasonable suggestion for now.

For context, though: I've done this at least twice, albeit special-cased in the rollup tool. When we switch to a better rollup/packaging tool we'll be able to properly dispense with this Javaism.